### PR TITLE
Ningjie

### PR DESCRIPTION
**Pull request by hunjhunj**
I updated the new demo doc. Please let me know if we will need to change anything!

### CODE REVIEW

File: docs/source/examples/TidalFlow2ACO/TidalFlow2ACO.rst [hunk 1]
```diff
@@ -1,240 +0,0 @@
...
-    tfc = TidalFlowCalculator(
-            grid,
-            tidal_period=tidal_period,
-            tidal_range=tidal_range,
-            roughness='roughness',
-            mean_sea_level=mean_sea_level,
-            min_water_depth=mwd,
-    )
-
-Assign critical shear stress (?)
```
Comment (ale37911) [inline docs/source/examples/TidalFlow2ACO/TidalFlow2ACO.rst:143]: yes

File: docs/source/examples/TidalFlow2ACO/TidalFlow2ACO.rst [hunk 2]
```diff
@@ -1,240 +0,0 @@
...
-
-Consider erosion
-----------------
-
-Set mud erodability
-~~~~~~~~~~~~~~~~~~~
-
-.. code:: ipython3
-
-    mud_erodability = (10**-5)*24*3600;  # mud erodability kg/m2/day!!!
```
Comment (ale37911) [inline docs/source/examples/TidalFlow2ACO/TidalFlow2ACO.rst:213]: ahh this should be changed to 10**-5 --> remove the 24*3600 so it's in kg/m2/s

File: docs/source/examples/TidalFlow2ACO/TidalFlow2ACO.rst [hunk 3]
```diff
@@ -1,240 +0,0 @@
...
-
-.. code:: ipython3
-
-    ero = tec.totalsedimenterosion_mudsine(grid, mud_erodability)
-    #ok so for some reason erosion value is SUPER high -- no clue why...need to clear that up!
-
-.. code:: ipython3
-
-    for i in range(2):
```
Comment (ale37911) [inline docs/source/examples/TidalFlow2ACO/TidalFlow2ACO.rst:222]: I've got an updated version of this for-loop - maybe I'm looking at an older version?

**Review by ale37911 — COMMENTED** [review]
this looks really good.

File: docs/source/examples/TidalFlow2ACO/TidalFlow2ACO.rst (hunk 3 above)
Comment (hunjhunj) [reply docs/source/examples/TidalFlow2ACO/TidalFlow2ACO.rst]: That's the old version. Let me close this one and do the PR again.

Comment (hunjhunj) [reply docs/source/examples/TidalFlow2ACO/TidalFlow2ACO.rst:222]: Yes, TidalFlow2ACO_formatted_V2.ipynb should be the new one

Comment (hunjhunj) [reply docs/source/examples/TidalFlow2ACO/TidalFlow2ACO.rst:222]: or TidalFlow2ACO_formatted_V2.rst (for sphinx)

**Pull request merged**